ibm_catalog.json

```diff
@@ -47,10 +47,385 @@
+              },
+              {
+                "title": " ",
+                "description": "Focuses on provisioning the IBM Cloud infrastructure around Code Engine—such as IAM, observability, storage, encryption, and security—while giving users the flexibility to integrate only what they need."
+              },
+              {
+                "title": " ",
+                "description": "Code Engine components (app, build, or function) are configured later by users via UI, CLI, or Terraform, offering full flexibility."
+              }
+            ],
+            "diagrams": [
+              {
+                "diagram": {
+                  "caption": "IBM Cloud Code Engine topology - Standard (Integrated setup with configurable services)",
+                  "url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-landing-zone-code-engine/refs/heads/quickstart_2/reference-architectures/deployment-architecture-fully-configurable.svg",
+                  "type": "image/svg+xml"
+                },
+                "description": "The architecture consists of a central [IBM Cloud Code Engine project](https://cloud.ibm.com/docs/codeengine?topic=codeengine-getting-started), which serves as the primary execution environment for the application. Surrounding this core component is the foundational IBM Cloud configuration, which includes the IBM Cloud account, resource group, account settings, and optionally a trusted profile for secure IAM access.<br><br>Encircling the Code Engine project are a set of optional service surrounds that can be integrated based on the specific use case. These include observability tools such as [Cloud Monitoring](https://www.ibm.com/products/cloud-monitoring), [Cloud Logs](https://www.ibm.com/products/cloud-logs), and Activity Tracker, which provide insight into application performance, logging, and user or system actions. Additional core services may also be incorporated, such as [Cloud Object Storage](https://www.ibm.com/products/cloud-object-storage) for data storage, [Key Protect](https://www.ibm.com/products/key-protect) for encryption key management, App Configuration for externalizing environment-specific settings, and [SCC Workload Protection](https://www.ibm.com/products/security-and-compliance-center) to enforce security and compliance policies at the workload level."
+              }
+            ]
+          },
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key"
+            },
+            {
+              "key": "prefix",
+              "required": true,
+              "value_constraints": [
+                {
+                  "type": "regex",
+                  "description": "Prefix must begin with a lowercase letter and may contain only lowercase letters, digits, and hyphens '-'. It must not end with a hyphen('-'), and cannot contain consecutive hyphens ('--'). It should not exceed 16 characters",
```

Comment:
This does not align with our prefix guidance documented in https://terraform-ibm-modules.github.io/documentation/#/da-implementation-guidelines?id=code-snippet-example